specs/dwritecore/DWriteCore.md

```diff
@@ -510,9 +510,217 @@ The following other DirectWrite APIs also use the hybrid font selection algorith
 -   `IDWriteTextLayout` if the typographic font family model is enabled by calling
     `IDWriteTextLayout4::SetFontAxisValues` or `IDWriteTextLayout4::SetAutomaticFontAxes`
 
-# Examples
-
-## Creating an isolated factory
+## Color Font Support
+
+By default, a glyph has a shape but no intrinsic color. Both DirectWrite and Direct2D have
+`DrawGlyphRun` methods that render glyph runs by filling the glyph shapes with a specified text
+color. For convenience, we will refer to this as _monochrome_ glyph rendering. All fonts have
+monochrome glyphs, but a color font also has color representations of some glyphs. To render glyphs
+in color, an application must use different glyph rendering APIs instead of calling the monochrome
+`DrawGlyphRun` method.
+
+Because color font support is implemented at the glyph rendering level, it does not affect text
+layout. This is true whether a client uses the `IDWriteTextLayout` interface or implements its own
+text layout algorithm. The mapping of characters to glyphs and the positioning of those glyphs all
+use monochrome glyph IDs and their associated metrics. The output of the text layout process is a
+sequence of monochrome glyph runs. Color font support can be enabled by translating those monochrome
+"base" glyph runs to color glyphs runs at rendering time.
+
+The easiest way to enable color is to call the `DrawGlyphRunWithColor` method instead of calling
+`DrawGlyphRun`. Direct2D exposes this method through the `ID2D1DeviceContext7` interface.
+DirectWrite exposes this method through the `IDWriteBitmapRenderTarget3` interface.
```

Comment:
DISCUSSION: These are not all equivalent - you need to provide palette information in some, and awesomeness levels to others. Fonts are backwards compatible; the awesomeness parameter indicates how to operate on a font that is able to be awesome.

The current shape means you need to pass "use awesome" and then "here is the awesome" ... if you don't want awesome, but you still want to call this method, you still need to pass in a bogus/unused awesome value.

The D2D API does take a color palette parameter, but it might not be used for various reasons (B&W font, image font, font without color hinting.)

RECOMMEND: DrawGlyphRunWithColorSupport - the "with" means it's extra, and not really an overload.